src/modals/publication/EditPublicationModal.vue

```diff
@@ -220,6 +235,33 @@ export default {
 					this.loading = false
 				})
 		},
+		fetchOrganizations() {
+			this.organizationsLoading = true
+
+			organizationStore.getAllOrganization()
+				.then(({ response, data }) => {
+					const selectedOrganization = data.filter((org) => org?.id.toString() === publicationStore.publicationItem?.organization.toString())[0] || null
+
+					this.organizations = {
+						options: data.map((organization) => ({
+							id: organization.id,
+							label: organization.title,
+						})),
+						value: selectedOrganization
+							? {
+								id: selectedOrganization?.id,
+								label: selectedOrganization?.title,
+							}
+							: null,
+					}
+
+					this.organizationsLoading = false
+				})
+				.catch((err) => {
+					console.error(err)
+					this.organizationsLoading = false
+				})
```

Comment:
Het liefst zouden we `.finally()` gebruiken om loading states op false te zetten